### PR TITLE
Download of available sample data

### LINKED ISSUE #13: Allow Manifest Download
**Describe the solution you'd like**
Make it possible to download a manifest file

### PR DESCRIPTION
**Pull request by KochTobi**
User Story: As a DA, I want to access a project and download all available sequencing raw data so that I can start analysing them

- [x] #53 
- [x] #57
- [x] #58 
- [x] #60
- [x] #61 
- [x] #62 
- [x] #74 
- [x] #76

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,11 +5,13 @@ Changelog
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 
-0.3.0-SNAPSHOT (2021-07-20)
+0.3.0-SNAPSHOT (2021-08-18)
 ---------------------------
 
 **Added**
 
+* Download of available samples is possible as a manifest file (`#54 <https://github.com/qbicsoftware/sample-tracking-status-overview/pull/54/conflicts>`_)
```
Comment (Steffengreiner) [inline CHANGELOG.rst:13]: Is there a reason why this links to to the conflicts subsection of the pull request?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,11 +5,13 @@ Changelog
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 
-0.3.0-SNAPSHOT (2021-07-20)
+0.3.0-SNAPSHOT (2021-08-18)
```
Comment (Steffengreiner) [inline CHANGELOG.rst:8]: Nit: The date should be updated to the date of this merge

**Review by Steffengreiner — CHANGES_REQUESTED** [review]
Nicely Done 🚀 
However i found an incorrect link in the Changelog, which should be changed. Once that's done i'll approve the PR 👍

File: CHANGELOG.rst (hunk 2 above)
Comment (KochTobi) [reply CHANGELOG.rst:8]: I don't think we need to do it here since we will do it in the release PR anyhow :D

Comment (Steffengreiner) [reply CHANGELOG.rst:8]: True, that's why i set it to Nit 👍

**Review by Steffengreiner — APPROVED** [review]
Nice!

**Pull request merged**